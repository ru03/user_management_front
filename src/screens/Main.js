import React from 'react';
import { Container } from '@material-ui/core';
import UserList from '../components/UserList';

const Main = () => (
    <Container>
      <UserList />
    </Container>
);

export default Main;