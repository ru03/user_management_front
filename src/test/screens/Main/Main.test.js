import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Main from '../../../screens/Main';

describe('Main Screen', () => {
  it('renders main screens', () => {
    const { asFragment } = render(<Main />, { wrapper: MemoryRouter });
    expect(asFragment()).toMatchSnapshot();
  });
});