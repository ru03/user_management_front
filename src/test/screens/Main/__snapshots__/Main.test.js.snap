// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Screen renders main screens 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
    >
      <div
        class="MuiBox-root MuiBox-root-113"
      >
        <div
          class="MuiPaper-root MuiTableContainer-root MuiPaper-elevation1 MuiPaper-rounded"
          mt="5"
        >
          <div
            class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
            role="progressbar"
            style="width: 40px; height: 40px;"
          >
            <svg
              class="MuiCircularProgress-svg"
              viewBox="22 22 44 44"
            >
              <circle
                class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
                cx="44"
                cy="44"
                fill="none"
                r="20.2"
                stroke-width="3.6"
              />
            </svg>
          </div>
          <div
            class="MuiTablePagination-root"
          >
            <div
              class="MuiToolbar-root MuiToolbar-regular MuiTablePagination-toolbar MuiToolbar-gutters"
            >
              <div
                class="MuiTablePagination-spacer"
              />
              <p
                class="MuiTypography-root MuiTablePagination-caption MuiTypography-body2 MuiTypography-colorInherit"
              >
                0-0 of 0
              </p>
              <div
                class="MuiTablePagination-actions"
              >
                <button
                  aria-label="Previous page"
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit Mui-disabled Mui-disabled"
                  disabled=""
                  tabindex="-1"
                  title="Previous page"
                  type="button"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      role="presentation"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M15.41 16.09l-4.58-4.59 4.58-4.59L14 5.5l-6 6 6 6z"
                      />
                    </svg>
                  </span>
                </button>
                <button
                  aria-label="Next page"
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit Mui-disabled Mui-disabled"
                  disabled=""
                  tabindex="-1"
                  title="Next page"
                  type="button"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      role="presentation"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M8.59 16.34l4.58-4.59-4.58-4.59L10 5.75l6 6-6 6z"
                      />
                    </svg>
                  </span>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
